bokehjs/src/lib/models/transforms/interpolator.ts

```diff
@@ -81,7 +81,7 @@ export abstract class Interpolator extends Transform {
       throw new Error("x and y must have at least two elements to support interpolation")
 
     const n = tsx.length
-    const index = new Float32Array(n)
+    const index = new NumberArray(n)
```

Comment:
The original type should have been `Uint32Array`. At least search and replace helped to uncover a bug.